content/2_Bronze/Intro_Sets.mdx

```diff
@@ -70,6 +65,17 @@ Unordered sets work with primitive types, but require a
 [custom hash function](/gold/hashmaps#custom-hashing)
 for structures/classes like vectors and pairs.
 
+<Warning title="Unordered Set Performance">
+
+`unordered_set` actually has worst-case $\mathcal{O}(N)$ behavior, and it's
+possible to construct a test case for Distinct Numbers such that a program
+that uses `unordered_set` times out.
```

Comment:
the test case was added, so change this to say that `unordered_set` times out on Distinct Numbers